src/olympia/devhub/tasks.py

```diff
@@ -140,22 +139,41 @@ def create_version_for_upload(addon, upload, channel):
         add_dynamic_theme_tag(version)
 
 
+@task
+def create_initial_validation_results():
+    """Returns the initial validation results for the next chains in the
+    validation tasks. Should only be called directly by Validator."""
+    results = deepcopy(amo.VALIDATOR_SKELETON_RESULTS)
+    return results
+
+
 def validation_task(fn):
-    """Wrap a validation task so that it runs with the correct flags, then
-    parse and annotate the results before returning."""
+    """Wrap a validation task so that it runs with the correct flags and
+    handles errors.
```

Comment:
Mention the signature the functions must have, and where `results` is coming from (it's not obvious if you haven't read `create_initial_validation_results()` before)